Makefile

```diff
@@ -0,0 +1,11 @@
+project = lisp
```

Comment:
What's this variable for?

Reply:
Oh I meant to remove that. Thanks for pointing it out.